bridge/client/app/project-board/project-board.component.html

```diff
@@ -166,3 +223,13 @@ <h2 class="m-0" [textContent]="service.serviceName"></h2>
     </div>
   </ng-container>
 </div>
+<ng-template #noDeployment>
+  <p class="m-0">Not deployed yet. Use <a href="https://keptn.sh/docs/0.6.0/reference/cli/commands/keptn_send_event_new-artifact/" target="_blank">keptn send new-artifact</a> to trigger a deployment.</p>
+</ng-template>
+<ng-template #noService>
```

Comment:
FYI Barista has a nice empty-state component for such use cases

Reply:
In this case, the empty state is displayed inside a card element and the empty-state component might not fit there.